android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt

```diff
@@ -161,6 +162,10 @@ open class AppMainActivity : BaseMultiLanguageActivity(), OnSyncListener {
       }
       is State.Finished -> {
         showToast(getString(R.string.sync_completed))
+        showToast(getString(R.string.sync_completed))
+        val totalTime = ChronoUnit.SECONDS.between(appMainViewModel.syncStartedAt, state.result.timestamp)
+        //showToast(getString(R.string.sync_completed) + " in "+totalTime + " seconds")
+        Timber.d(getString(R.string.sync_completed) + " in "+totalTime + " seconds")
```

Comment:
Was this used for debugging?

Reply:
Can we have this in another PR if this functionality is needed.

Reply:
ok